Files/Views/PaneHolderPage.xaml

```diff
@@ -114,11 +114,11 @@
                 <local:ModernShellPage
                     x:Name="PaneLeft"
                     ContentChanged="Pane_ContentChanged"
-                    CurrentInstanceBorderThickness="0,0,2,0"
+                    CurrentInstanceBorderThickness="0,0,0,2"
```

Comment:
Is there any reason in particular you changed which side has the border?

Reply:
I find it difficult to tell what pane is active when the border is between the two panes. Since the issue author also states the same, I propose to move the border at the bottom.